doc/fluid/dev/api_doc_std_cn.md

```diff
@@ -137,7 +137,7 @@ API文档各模块格式及示例如下(以下以fc为例进行说明）：
 
       ```
       Args:
-          input (Tensor): The input tensor(s) of the layer.
+          input (Variable|list of Variable): This layer's input tensor(s) which is at least 2-dimensional.
```

Comment:
The input tensor(s) of the layer, and the dimension of the input tensor(s) is at least 2.